src/registry.ts

```diff
@@ -0,0 +1,3 @@
+import config from 'config';
+
+export const prefix = `${config.get('SchemaRegistry.Protocol')}://${config.get('SchemaRegistry.Hostname')}:${config.get('SchemaRegistry.Port')}/apis/registry`;
```

Comment:
Can you please uppercase the constant to make it more clear when it's being used?

Reply:
Done